docs/architecture/ADR002.md

```diff
@@ -0,0 +1,33 @@
+# ADR002: Decouple what we're calculating bills for from how the bills are calculated
```

Comment:
Is this what it means, or is it more than whether or not they're already a tenant?
```suggestion
# ADR002: Decouple how the bills are calculated from whose bills they are
```